### PR TITLE
updated Game.java based off sprint2 repo

### PR DESCRIPTION
**Pull request by kevinkoos**
I've modified the card removal function and Game constructor based off the sprint2 repo.

### CODE REVIEW

File: src/main/java/models/Game.java [hunk 1]
```diff
@@ -17,8 +17,7 @@
     public Game(){
         // initialize a new game such that each column can store cards
         for (int i = 0; i < 4; i++) {
-            java.util.List<Card> tmpList = new ArrayList<>(42);
-            cols.add(i,tmpList);
+            cols.add(i,new ArrayList<Card>());
```
Comment (mgray35) [inline src/main/java/models/Game.java]: Yeah, this looks much cleaner

File: src/main/java/models/Game.java [hunk 2]
```diff
@@ -46,9 +45,26 @@ public void dealFour() {
...
+                        if (compare.getSuit() == c.getSuit()) {
+                            if (compare.getValue() > c.getValue()) {
+                                removeCard = true;
+                            }
+                        }
+                    }
+                }
+            }
+            if (removeCard) {
+                this.cols.get(columnNumber).remove(this.cols.get(columnNumber).size() - 1);
```
Comment (mgray35) [inline src/main/java/models/Game.java]: Can you explain what this line does? There are a lot of dots and I'm having trouble following it

Comment (mgray35) [inline src/main/java/models/Game.java]: Can you explain what this line does? There are a lot of dots and I'm having trouble following it

Comment (kevinkoos) [reply src/main/java/models/Game.java]: if removeCard (bool saying that the removal is valid) is true
cols.get(colunmNumber) gets the specific column(arraylist) of the card to be removed
*.remove(***) says to remove the the last card from the column where *** is the code the get that last card

**Pull request merged**